org.eclipse.jdt.core/search/org/eclipse/jdt/internal/core/search/indexing/AddJarFileToIndex.java

```diff
@@ -92,7 +92,7 @@ public int hashCode() {
 	@Override
 	public boolean execute(IProgressMonitor progressMonitor) {
 
-		if (this.isCancelled || progressMonitor != null && progressMonitor.isCanceled()) return true;
+		if (this.isCancelled || ((progressMonitor != null) && progressMonitor.isCanceled())) return true;
```

Comment:
Is this an auto-formatting result? Syntactically, the brackets are unnecessary and at least the ones put around the progressMonitor check reduce readability.

Reply:
I was an auto-formatting setting, thank you. I reverted these changes